### PR TITLE
New package: cloudflared-2022.7.1

### PR DESCRIPTION
**Pull request by marmeladema**

### CODE REVIEW

File: srcpkgs/cloudflared/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=cloudflared
+version=2021.7.0
+revision=1
+build_style=go
+go_import_path=github.com/cloudflare/cloudflared
+go_package="${go_import_path}/cmd/cloudflared"
+go_ldflags="-X \"main.Version=${version}\" -X \"main.BuildTime=$(date -u '+%Y-%m-%d-%H:%M UTC')\""
+short_desc="Argo Tunnel client"
+maintainer="Elie ROUDNINSKI <xademax@gmail.com>"
+license="custom:cloudflare-servicesagreement"
```
Comment (Piraty) [inline srcpkgs/cloudflared/template:11]: [this](https://github.com/cloudflare/cloudflared/blob/master/LICENSE#L37-L41) read pretty nonfree to me

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/cloudflared/template (hunk 1 above)
Comment (marmeladema) [reply srcpkgs/cloudflared/template:11]: And?

Comment (marmeladema) [reply srcpkgs/cloudflared/template:11]: Done :+1:

**Comment by Piraty** [comment]
upstream switched to apache license

File: srcpkgs/cloudflared/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'cloudflared'
+pkgname=cloudflared
+version=2021.7.0
+revision=1
+build_style=go
+go_import_path=github.com/cloudflare/cloudflared
+go_package="${go_import_path}/cmd/cloudflared"
+go_ldflags="-X \"main.Version=${version}\" -X \"main.BuildTime=$(date -u '+%Y-%m-%d-%H:%M UTC')\""
```
[the remark this answers is not in the document]
Comment (marmeladema) [reply srcpkgs/cloudflared/template:8]: What would be the correct approach then?

**Comment by marmeladema** [comment]
@Piraty updated the template to reflect the new license

File: srcpkgs/cloudflared/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (Piraty) [reply srcpkgs/cloudflared/template:8]: `srcpkgs/docker-cli/template` does this too...

[the remark this answers is not in the document]
Comment (Piraty) [reply srcpkgs/cloudflared/template:8]: @marmeladema: make use of `$SOURCE_DATE_EPOCH` instead

[the remark this answers is not in the document]
Comment (marmeladema) [reply srcpkgs/cloudflared/template]: Sorry @Piraty, I wasn't sure exactly what is the value of `$SOURCE_DATE_EPOCH` so I decided to go for the docker-cli approach. I hope that's ok!

[the remark this answers is not in the document]
Comment (Piraty) [reply srcpkgs/cloudflared/template]: https://reproducible-builds.org/docs/source-date-epoch/

[the remark this answers is not in the document]
Comment (marmeladema) [reply srcpkgs/cloudflared/template]: @Piraty is my current approach still a blocker then? I have used date *inside* a pre_ function so it should use `SOURCE_DATE_EPOCH` implicitly

File: srcpkgs/cloudflared/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'cloudflared'
+pkgname=cloudflared
+version=2022.7.1
+revision=1
+build_style=go
+go_import_path=github.com/cloudflare/cloudflared
+go_package="${go_import_path}/cmd/cloudflared"
+go_ldflags="-X \"main.Version=${version}\""
```
Comment (paper42) [inline srcpkgs/cloudflared/template:8]:
```suggestion
go_ldflags="-X main.Version=${version}"
```
this should also work fine

**Comment by Piraty** [comment]
broken. 02c4c21227

**Pull request merged**